chunkSize is 0 when table is empty

## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do?
If possible, provide a recipe for reproducing the error.
compare 2 empty table

2. What did you expect to see?
see result is success

3. What did you see instead?
panic: runtime error: integer divide by zero

4. What version of TiDB are you using (`tidb-server -V` or run `select tidb_version();` on TiDB)?
v5.3.0

5. which tool are you using?
```
[root@copy-of-vm-ee-centos76-v1 bin]# ./sync_diff_inspector --config configa.toml 
A total of 1 tables need to be compared

Progress [>------------------------------------------------------------] 0% 0/0
panic: runtime error: integer divide by zero

goroutine 78 [running]:
github.com/pingcap/tidb-tools/sync_diff_inspector/splitter.NewRandomIteratorWithCheckpoint(0x253cd18, 0xc00015e280, 0xc0009c5cf0, 0xc, 0xc000543300, 0xc00084d450, 0x0, 0x4, 0x0, 0x2502b80)
```
6. what versionof tool are you using (`pump -V` or `tidb-lightning -V` or `syncer -V`)?

--- a/sync_diff_inspector/splitter/random.go
+++ b/sync_diff_inspector/splitter/random.go
@@ -102,7 +102,9 @@ func NewRandomIteratorWithCheckpoint(ctx context.Context, progressID string, tab
 				// no index
 				// will use table scan
 				// so we use one chunk
-				chunkSize = cnt
+				// plus 1 to avoid chunkSize is 0
+				// while chunkCnt = (2cnt)/(cnt+1) <= 1
+				chunkSize = cnt + 1
 			}
 		}
 		log.Info("get chunk size for table", zap.Int64("chunk size", chunkSize),

--- a/sync_diff_inspector/splitter/splitter_test.go
+++ b/sync_diff_inspector/splitter/splitter_test.go
@@ -898,7 +898,7 @@ func TestChunkSize(t *testing.T) {
 	createFakeResultForRandomSplit(mock, 1000, nil)
 	randomIter, err = NewRandomIterator(ctx, "", tableDiff_noindex, db)
 	require.NoError(t, err)
-	require.Equal(t, randomIter.chunkSize, int64(1000))
+	require.Equal(t, randomIter.chunkSize, int64(1001))
 
 	// test limit splitter chunksize
 	createFakeResultForCount(mock, 1000)

